P1/index.html

```diff
@@ -1,14 +1,134 @@
-<!DOCTYPE html>
```

Comment:
unnoticable on the current background but still exists
![image](https://user-images.githubusercontent.com/2150198/28066224-caeb93ea-6644-11e7-90f4-050a43ff79fd.png)